shared/actions/chat.js

```diff
@@ -401,13 +407,61 @@ function * _openFolder (): SagaGenerator<any, any> {
 
 function * _newChat (action: NewChat): SagaGenerator<any, any> {
   yield put(searchReset())
-  yield put(searchAddUsersToGroup(action.payload.existingParticipants.map(usernameToSearchResult)))
+
+  const metaDataSelector = (state: TypedState) => state.chat.get('metaData')
+  const metaData = ((yield select(metaDataSelector)): any)
+
+  const following = (yield select(followingSelector)) || {}
+
+  yield put(searchAddUsersToGroup(action.payload.existingParticipants.map(username => ({
+    service: 'keybase',
+    username,
+    isFollowing: !!following[username],
+    extraInfo: {
+      service: 'none',
```

Comment:
We're essentially making a fake search result and only the usernames are really used. I think later we could potentially make a new api endpoint to get the exact search results for all participants but seems like not an MVP thing at the moment.